VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -454,8 +461,10 @@ public void testUnitFragmentCreation() {
     // TODO: Robolectric doesn't have a Gradle-based parameterized test suite, need to implement it.
     @Ignore
     @RunWith(Parameterized.class)
-    public static class ParameterizedFragmentCreation {
-        @Parameterized.Parameters(name= "{index}: testUnitFragmentCreation({0})={1}")
+    public static class ParameterizedFragmentCreation extends UiTest {
+        private static final String DISCUSSIONS_ENABLED = "DISCUSSIONS_ENABLED";
```

Comment:
It would probably be better to create a mock config class that allows setting properties directly, but we can consider that out of scope for now I guess.